cves/CVE-2016-8747.yml

```diff
@@ -10,27 +10,27 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: CWE-200
```

Comment:
Change this to just the number 200, there should be nothing except the number of the CWE.

Reply:
I just updated it to only have 200